spec

```diff
@@ -0,0 +1,12532 @@
+
+    include "build/asm/boot/getsr.text.o"
+    include "build/asm/boot/setsr.text.o"
+
+    include "build/src/boot/writebackdcache.o"
+
+    include "build/data/boot/threadsave.bss.o"
+
+    include "build/src/libultra/os/initialize.o"
+    include "build/data/boot/initialize.bss.o"
+
+    pad_text
+    pad_text
+    pad_text
+    pad_text
+    pad_text
+    pad_text
+
+    include "build/src/libultra/io/vigetnextframebuf.o"
+
+    include "build/src/libultra/gu/perspective.o"
+
+    include "build/src/libultra/io/sprawdma.o"
+    include "build/src/libultra/io/sirawdma.o"
+    include "build/src/boot/epilinkhandle.o"
+    include "build/src/libultra/io/viblack.o"
+    include "build/src/libultra/io/sirawread.o"
+    include "build/src/libultra/os/getthreadid.o"
+    include "build/src/libultra/io/sptaskyield.o"
+    include "build/src/boot/pfsreadwritefile.o"
+    include "build/src/boot/pfsgetstatus.o"
+    pad_text
+    include "build/asm/boot/guMtxIdentF.text.o"
+    include "build/src/libultra/io/visetmode.o"
+
+    include "build/asm/boot/getconfig.text.o"
+    include "build/asm/boot/setconfig.text.o"
+
+    include "build/src/libultra/gu/lookat.o"
+    include "build/src/boot/pfsallocatefile.o"
+    include "build/src/libultra/os/stoptimer.o"
+
+    include "build/asm/boot/probetlb.text.o"
+
+    include "build/src/libultra/io/pimgr.o"
+    include "build/data/boot/pimgr.data.o"
+    include "build/data/boot/pimgr.bss.o"
+
+    include "build/src/libultra/io/piacs.o"
+
+    pad_text
+    include "build/src/boot/devmgr.o"
+
+    include "build/src/libultra/io/pirawdma.o"
+
+    include "build/src/boot/contpfs.o"
+    include "build/data/boot/contpfs.data.o"
+
+    include "build/asm/boot/getcount.text.o"
+
+    pad_text
+
+    include "build/asm/boot/guMtxL2F.text.o"
+    include "build/src/libultra/os/getmemsize.o"
+    include "build/src/boot/pfssearchfile.o"
+
+    include "build/src/libultra/os/seteventmesg.o"
+    include "build/data/boot/seteventmesg.bss.o"
+
+    include "build/src/boot/sqrtf.o"
+    include "build/src/boot/afterprenmi.o"
+    include "build/src/libultra/io/contquery.o"
+
+    include "build/src/libultra/gu/lookathil.o"
+
+    include "build/src/libultra/rmon/xprintf.o"
+
+    include "build/src/boot/voicecleardictionary.o"
+    include "build/asm/boot/unmaptlball.text.o"
+    include "build/src/libultra/io/epidma.o"
+    include "build/src/boot/voicecontread2.o"
+    include "build/src/boot/voicecrc.o"
+    include "build/src/libultra/libc/string.o"
+    include "build/src/libultra/os/createmesgqueue.o"
+    include "build/src/boot/invalicache.o"
+    include "build/src/boot/invaldcache.o"
+
+    include "build/src/boot/timerintr.o"
+    include "build/data/boot/timerintr.data.o"
+    include "build/data/boot/timerintr.bss.o"
+    /*
+    include "build/src/libultra/os/timerintr.o"
+    //include "build/data/boot/timerintr.data.o"
+    //include "build/data/boot/timerintr.bss.o"
+    */
```

Comment:
Is this still meant to be here?

Reply:
The commented file should be the one included in the spec. The thing is, the file wouldn't produce an OK in any way I tried, so I added the other one as a workaround in the meanwhile.
If somebody wants to try and match the one in the libultra folder, go for it!

Reply:
I've PR'd a fix for this to the branch: there was some weird extra space in the bss I had to pad for, at least for now.